### PR TITLE
Fix: Bugs of report in common report filter apply check of role permissions

### PR DESCRIPTION
**Pull request by abhirock74**

### CODE REVIEW

File: myojana/myojana/report/total_approved_demands/total_approved_demands.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ def execute(filters=None):
         }
     ]
 
-    condition_str = ReportFilter.set_report_filters(filters, 'date_of_visit', True)
+    condition_str = ReportFilter.set_report_filters(filters, 'creation', True)
     if condition_str:
-        condition_str = f"AND {condition_str}"
+        condition_str = f"AND {condition_str} AND 1=1"
```
Comment (amreshyadavgit) [inline myojana/myojana/report/total_approved_demands/total_approved_demands.py:22]: Relook into this code and test it.

**Review by amreshyadavgit — CHANGES_REQUESTED** [review]
Resolve the comments

**Comment by abhirock74** [comment]
Creating new Pr

File: myojana/myojana/report/total_approved_demands/total_approved_demands.py (hunk 1 above)
Comment (abhirock74) [reply myojana/myojana/report/total_approved_demands/total_approved_demands.py:22]: Hi @amreshyadavgit ,
Thanks for you suggestion .
As per your suggestion I have change the code and test please look into it and verify if possible

File: myojana/myojana/report/age_wise_beneficiaries/age_wise_beneficiaries.py [hunk 2]
```diff
@@ -22,13 +22,16 @@ def execute(filters=None):
         }
     ]
     condition_str = ReportFilter.set_report_filters(
-        filters, 'creation', True)
+        filters, 'date_of_visit', True)
+    print ("condition_str", condition_str)
```
Comment (amreshyadavgit) [inline myojana/myojana/report/age_wise_beneficiaries/age_wise_beneficiaries.py:26]: remove/comment print before commit.

File: myojana/myojana/report/beneficiaries_who_have_received_benefits_of_3_or_more_schemes/beneficiaries_who_have_received_benefits_of_3_or_more_schemes.py [hunk 3]
```diff
@@ -25,7 +25,7 @@ def execute(filters=None):
 	]
 	condition_str = ReportFilter.set_report_filters(filters, 'creation', True)
 	if condition_str:
-		condition_str = f"AND {condition_str}"
+		condition_str = f"AND {condition_str} 1=1"
```
Comment (amreshyadavgit) [inline myojana/myojana/report/beneficiaries_who_have_received_benefits_of_3_or_more_schemes/beneficiaries_who_have_received_benefits_of_3_or_more_schemes.py:28]: this is repeated mistake, fix it.

**Review by amreshyadavgit — CHANGES_REQUESTED** [review]

File: myojana/myojana/report/age_wise_beneficiaries/age_wise_beneficiaries.py (hunk 2 above)
Comment (abhirock74) [reply myojana/myojana/report/age_wise_beneficiaries/age_wise_beneficiaries.py]: Hi @amreshyadavgit ,
It is Done

File: myojana/myojana/report/beneficiaries_who_have_received_benefits_of_3_or_more_schemes/beneficiaries_who_have_received_benefits_of_3_or_more_schemes.py (hunk 3 above)
Comment (abhirock74) [reply myojana/myojana/report/beneficiaries_who_have_received_benefits_of_3_or_more_schemes/beneficiaries_who_have_received_benefits_of_3_or_more_schemes.py]: Hii @amreshyadavgit ,
It is Done

**Review by abhirock74 — COMMENTED** [review]
Done @amreshyadavgit ,
Please Review once again

**Review by amreshyadavgit — APPROVED** [review]

**Pull request merged**